### ISSUE TITLE
Web server not responding anymore in V3.6.0

### ISSUE THREAD

**Issue by RonaldSpijk** [dingo35/SmartEVSE-3.5#79]
I've uploaded version 3.6.0 on a newly bought SmartEVSE. Before that it had 3.5.1 for a few days.
It worked for a few days without any problems.
 
After some issues with the Wifi the web server stopped working. MQTT still works.
I can also ping.
When accessing port 80 the port is found, but the connection is dropped immediately.

A tried resetting a few times, doesn't help.
A tried starting the Wifi Portal, this also doesn't work.

Fortunately, charging still works :-)

I'm stuck in the middle here, because when the web server doesn't work, I can't update,  and get logs.
I'm afraid to do anything else, because it might brick my SmartEVSE.

I think this needs good investigation because it might also occur at other users.
And I like some help getting a working version on the SmartEVSE. Would uploading the bin file with the Espressif flash download tool work? Or should I try something else first.

**Comment by AndrejSNC** [comment]
What I have discovered today is, I replaced my old SmartEVSE version with V3 controller. It came with 3.0.1 SW version. I connected it to WiFi and set fixed IP address - works great. 
Then I updated it to 3.6.0 it connected back to the old dynamic IP address, which I don't like. I can start WiFi setup in menu, connect to access point but the configuration page does not open on my phone. How can I access this configuration, to set back static IP? Tried 192.168.4.1 and it did not connect.

**Comment by dingo35** [comment]
If you set "wifi" from "Enabled" to "Setup" on the LCD screen, you should be able to connect to the SmartEVSE AP via 192.168.4.1 and set a static IP.

If it for some reason that fails, you can always goto /erasesettings and repeat the wifi setup procedure.

**Comment by deekoowee** [comment]
Similar problem here. Controlled worked until I upgrade it to 3.6.0
Controller answer for ping at 192.168.4.1, but do not answer to browser. Tried with Chrome and Curl
Also tried the /erasesettings, but no luck.
I tried to flash the firmware with **ESP32 flash download tool v3.9.6** , but nothing changed
Connected serial cable and telnet shows;
![pic](https://github.com/dingo35/SmartEVSE-3.5/assets/121685906/4f06856d-7d35-418e-8c7b-695fd9097be4)
Any suggestions to fix the controller's firmware

**Comment by dingo35** [comment]
If you have these problems with 3.6.0, why dont you downgrade to 3.5.1 with USB-C cable? Link is posted in this thread....

**Comment by deekoowee** [comment]
Downgraded to 3.5.1 via USB-C cable, rebooted controller, the web page showup.
Showed version 3.6.0 at left top corner of the screen
After few minutes, the webserver has lost, no communication with any browser.
Some mystery, which I don't catch up.

**Comment by dingo35** [comment]
Your downgrade obviously did not succeed...

**Comment by deekoowee** [comment]
I tried to upload the firmware.bin file with ESP32 flash download tool v3.9.6 (downloaded from [Espressif site](https://www.espressif.com/en/support/download/other-tools))
I used settings like this, only filename was changed.
![pic](https://github.com/dingo35/SmartEVSE-3.5/assets/121685906/d7d5c164-6b93-45df-a733-f134dbbc97be)
the bar at bottom on window runs ok.

**Comment by mstegen** [comment]
There are two partitions, the first one starts at 0x10000 
The second one starts at 0x1c0000 
Try flashing both partitions. That should do the trick.

**Comment by deekoowee** [comment]
Flashing those two partitions together fixed the webaccess problem

All settings were gone and have to reconfigure the controller

**Comment by dingo35** [comment]
The settings were gone because you did /erasesettings, not because of reflashing those two partitions...

**Comment by deekoowee** [comment]
Yes, I noticed that.
It is not big deal to make new settings again

**Comment by dingo35** [comment]
This should solve the problem:
[83adad145a90903.zip](https://github.com/user-attachments/files/15512629/83adad145a90903.zip)

**Comment by AlbertHakvoort** [comment]
> This should solve the problem: [83adad145a90903.zip](https://github.com/user-attachments/files/15512629/83adad145a90903.zip)

I've had the same issue, the 83adad145a90903.zip is fixed the webserver issue

**Comment by dingo35** [comment]
Thanks for your reports guys, this should be solved in 3.6.1 .

**Comment by Wireheadbe** [comment]
Mine "bricked" when updating from 3.7.0 to 3.7.2.
Here's how I flashed in a Debian-based distribution:

First, check where the device has popped up.. from experience, it's quite easy to find the last connected device by just looking into the kernel log, and looking in the last couple of lines:
```
sudo dmesg -T
```

Then, install usbtool, and flash the firmware:
```
sudo apt install esptool
esptool --port /dev/ttyUSB0 write_flash 0x10000 firmware.bin
esptool --port /dev/ttyUSB0 write_flash 0x1c0000 firmware.bin
```

It'll take some time to flash to both partitions, so don't stress :-)
